app/src/main/java/is/xyz/mpv/SettingsActivity.kt

```diff
@@ -0,0 +1,224 @@
+package `is`.xyz.mpv
+
+
+import android.annotation.TargetApi
+import android.content.Context
+import android.content.res.Configuration
+import android.os.Build
+import android.os.Bundle
+import android.preference.ListPreference
+import android.preference.Preference
+import android.preference.PreferenceActivity
+import android.app.ActionBar
+import android.preference.PreferenceFragment
+import android.view.MenuItem
+
+/**
+ * A [PreferenceActivity] that presents a set of application settings. On
+ * handset devices, settings are presented as a single list. On tablets,
+ * settings are split by category, with category headers shown to the left of
+ * the list of settings.
+ *
+ *
+ * See [
+   * Android Design: Settings](http://developer.android.com/design/patterns/settings.html) for design guidelines and the [Settings
+   * API Guide](http://developer.android.com/guide/topics/ui/settings.html) for more information on developing a Settings UI.
+ */
+class SettingsActivity : PreferenceActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setupActionBar()
+    }
+
+    /**
+     * Set up the [android.app.ActionBar], if the API is available.
+     */
+    private fun setupActionBar() {
+        val actionBar = actionBar
+        actionBar?.setDisplayHomeAsUpEnabled(true)
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    override fun onIsMultiPane(): Boolean {
+        return isXLargeTablet(this)
```

Comment:
Is it <i>normal</i> (in Android Dev) to only have multi pane on x-large tablet devices?

If not, it would make sense to document why this choice was made.